### PR TITLE
Add informal words to language features

### PR DESCRIPTION
**Pull request by Ladsgroup**

### CODE REVIEW

**Comment by halfak** [comment]
Looks great!  Thanks.

File: revscoring/languages/english.py [hunk 1]
```diff
@@ -35,15 +35,30 @@
     "yank", "yankee", "yid",
     "zipperhead"
 ])
+INFORMAL_WORDS = set([
+    'awesome', 'awesomest', 'awsome'
```
Comment (he7d3r) [inline revscoring/languages/english.py]: It is missing a "," at the end of a few lines. It seems this causes some concatenation in Python (e.g. "awsomebla", "bullshitcant")

File: revscoring/languages/english.py [hunk 2]
```diff
@@ -51,6 +66,14 @@ def is_badword(word):
 is_badword = LanguageUtility("is_badword", is_badword_process, depends_on=[stem_word])
 
 
+def is_informal_word_process(stem_word):
+    def is_informal_word(word):
+        return stem_word(word) in INFORMAL_WORDS
```
Comment (he7d3r) [inline revscoring/languages/english.py]: The INFORMAL_WORDS are not stemmed; (e.g. "boobs"  ->  "boob")

File: revscoring/languages/english.py [hunk 3]
```diff
@@ -35,15 +35,30 @@
...
+    "dont", "dumb", "dumbass",
+    "haha", "hello", "hey",
+    "kool",
+    "lol", "luv",
+    "meow",
+    'shove', 'smelly', 'sooo', 'stinky', 'sucking', 'sux'
+    "tits",
+    "wuz",
+    'yall', 'yay', 'yea', 'yolo'])
 STEMMED_BADWORDS = set(STEMMER.stem(w) for w in BADWORDS)
```
Comment (he7d3r) [inline revscoring/languages/english.py]: Create a similar variable for the STEMMED_INFORMAL_WORDS .

**Pull request merged**